servicedirectory/api/ServiceDirectory.Samples.Tests/ServiceDirectoryFixture.cs

```diff
@@ -0,0 +1,96 @@
+/*
+* Copyright 2020 Google LLC
+*
+* Licensed under the Apache License, Version 2.0 (the "License");
+* you may not use this file except in compliance with the License.
+* You may obtain a copy of the License at
+*
+*     https://www.apache.org/licenses/LICENSE-2.0
+*
+* Unless required by applicable law or agreed to in writing, software
+* distributed under the License is distributed on an "AS IS" BASIS,
+* WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+* See the License for the specific language governing permissions and
+* limitations under the License.
+*/
+
+using Google.Api.Gax.ResourceNames;
+using Google.Cloud.ServiceDirectory.V1Beta1;
+using System;
+using Xunit;
+
+[CollectionDefinition(nameof(ServiceDirectoryFixture))]
+public class ServiceDirectoryFixture : IDisposable, ICollectionFixture<ServiceDirectoryFixture>
+{
+    public string ProjectId { get; }
+    public ProjectName ProjectName { get; }

```

Comment:
```cs
public IList<string> TempNamespanceIds { get; } = new List<string>();
```
Please add List of Namespaces to remove that has been created in test and delete all in `Dispose` in Fixture with proper exception handling.